versions-check.csharp

```diff
@@ -68,6 +69,20 @@ try {
 			Console.Error.WriteLine ($"Could not find the min {product} version {min} in {Path.GetFileName (plistPath)}.");
 			failed = true;
 		}
+
+		var supportedTPVNodes = doc.SelectNodes ($"/plist/dict/key[text()='SupportedTargetPlatformVersions']/following-sibling::dict[1]/key[text()='{product}']/following-sibling::array[1]/string").Cast<XmlNode> ().ToArray ();
+		var supportedTPVs = supportedTPVNodes.Select (v => v.InnerText).ToArray ();
```

Comment:
I've started getting used to do the following:
```suggestion
		var supportedTPVs = supportedTPVNodes.Select (v => v.InnerText).ToArray () ?? Array.Empty<RolfIHaveNotIdeaOfTheType>();
```
to make my life easier later with the nullability.

Reply:
This is a C# script, so we can't do much about nullability warnings (or the lack thereof) - you'll just get an NRE if you get it wrong.